kong/cmd/utils/nginx_signals.lua

```diff
@@ -52,16 +52,15 @@ function _M.find_bin()
     local path_to_check = pl_path.join(path, nginx_bin_name)
     if is_openresty(path_to_check) then
       found = path_to_check
+      log.verbose("found OpenResty 'nginx' executable at %s", found)
       break
     end
   end
 
   if not found then
-    return nil, "could not find OpenResty 'nginx' executable"
+    return nil, ("could not find OpenResty 'nginx' executable. Kong requires version %s"):format(tostring(nginx_compatible))
```

Comment:
So, when an incompatible version will be found, the CLI will output twice the same information:
```
log.verbose("incompatible nginx found at "..bin_path..". Kong requires OpenResty, version "..tostring(nginx_compatible) 
-- [...]
return nil, ("could not find OpenResty 'nginx' executable. Kong requires version %s"):format(tostring(nginx_compatible))
```

Shouldn't we remove the verbose call, and throw a proper error from `is_openresty()`? So:
- is_openresty() returns `found, err`. The only possible error is "incompatible"
- after the is_openresty() call, `if not found`, then the `nginx` executable simply isn't in the path.

In both cases we return proper errors (and exit the CLI), and we don't duplicate any error message?